app/models/comments-model.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../../db/connection')
+
+exports.selectCommentsByArticleID = (articleID)=>{
+    const queryStr = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`
+    return db.query(queryStr, [articleID])
+    .then(({rows})=>{
```

Comment:
We want to be checking the the exact reason we are returning an empty row in this case.
This could be due to the following...
article exists but has no comments
OR
article does not exist.
We need to establish which is the case, and send appropriate error.  You will need to check whether the article exists in order to determine this.